models/gif_cond_exp.h

```diff
@@ -0,0 +1,450 @@
+/*
+ *  gif_cond_exp.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GIF_COND_EXP_H
+#define GIF_COND_EXP_H
+
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// Includes from nestkernel:
+#include "event.h"
+#include "archiving_node.h"
+#include "ring_buffer.h"
+#include "connection.h"
+
+#include "universal_data_logger.h"
+
+// Includes from gnu gsl:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+#include "nest.h"
+
+/* BeginDocumentation
+  Name: gif_cond_exp - Conductance based generalized integrate-and-fire neuron
+  model according to Mensi et al. (2012) and Pozzorini et al. (2015).
+
+  Description:
+
+  gif_psc_exp is the generalized integrate-and-fire neuron according to
+  Mensi et al. (2012) and Pozzorini et al. (2015), with post-synaptic
+  conductances in the form of truncated exponentials.
+
+  This model features both an adaptation current and a dynamic threshold for
+  spike-frequency adaptation. The membrane potential (V) is described by the
+  differential equation:
+
+  C*dV(t)/dt = -g_L*(V(t)-E_L) - eta_1(t) - eta_2(t) - ... - eta_n(t) + I(t)
+
+  where each eta_i is a spike-triggered current (stc), and the neuron model can
```

Comment:
Could you clarify the meaning of the ``eta`` name? I think this might help users understand the model more easily (I guess there might be something like "exponential" and "adaptation" involved, but I did not find it in the papers...)